packages/wonder-blocks-dropdown/components/dropdown-core.js

```diff
@@ -38,33 +43,98 @@ type DropdownCoreProps = {
 };
 
 export default class DropdownCore extends React.Component<DropdownCoreProps> {
+    node: ?HTMLDivElement;
+    // TODO(sophie): figure out flow typing
+    handleClick: (event: any) => void;
```

Comment:
The `any` can be a `SyntheticMouseEvent<"click">` I think.  See https://www.saltycrane.com/flow-type-cheat-sheet/latest/.

Reply:
I see, you have it defined below... you shouldn't have to define it here as well.